### PR TITLE
Changes for lbry-desktop

### PR DESCRIPTION
**Pull request by seanyesmunt**
My editor made a few formatting changes. I can revert them if you want.

Might need a few more, but everything seems to work well now. I will probably make another PR after https://github.com/lbryio/lbry-desktop/pull/2279 is merged. This is to just get it working.

### CODE REVIEW

File: sass/form/_index.scss [hunk 1]
```diff
@@ -111,18 +114,36 @@ input-submit {
...
+        border-bottom-color: $lbry-black;
+        border-left-color: transparent;
+      }
+
+      &:focus {
+        border-top-color: $lbry-teal-5;
+        border-right-color: $lbry-teal-5;
+        border-bottom-color: $lbry-teal-5;
+        border-left-color: transparent;
+      }
```
Comment (seanyesmunt) [inline sass/form/_index.scss]: To allow the button to be on the left or right.

File: sass/form/_index.scss [hunk 2]
```diff
@@ -1,23 +1,33 @@
...
+  [type="password"] {
     padding-right: var(--spacing-s);
     padding-left: var(--spacing-s);
     transition: border 0.2s;
   }
 
-  [type="button"],
-  [type="submit"] {
+  [type="submit"],
+  .button--primary {
```
Comment (seanyesmunt) [inline sass/form/_index.scss]: So we can have links/inverse buttons in the form for minor stuff.

Comment (NetOperatorWibby) [reply sass/form/_index.scss]: 👍🏾

**Comment by NetOperatorWibby** [comment]
Looks good to me

**Pull request merged**